_includes/distribution.html

```diff
@@ -49,7 +49,11 @@ <h1 class="display-3">
 <div class="tab-content">
   <div class="tab-pane active" id="overview" role="tabpanel">
 
-    {% include overview/{{ distro.name }}.html %}
```

Comment:
You could probably just do [remove filter](https://shopify.github.io/liquid/filters/remove/), with `" "` as the argument

Reply:
Worked, thank you for this one! I've tried my luck with gsub and whitespaces, but somehow I couldn't get that working (perhaps it was caused by same issue as the other includes that were raising value error)